contrib/add_license_to_files.jl

```diff
@@ -59,6 +59,7 @@ const skipfiles = [
     "../src/support/utf8.c",
     "../test/perf/micro/randmtzig.c",
     "../src/support/crc32c.c",
+    "../examples/quine.jl", # has license text in code
```

Comment:
It's a bit silly to have this in the quine example. It would probably be better to exclude it and remove the license text from that file.